camel/agents/tool_agents/Google_tool_agent.py

```diff
@@ -0,0 +1,56 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import os
+from typing import Any, Optional
+
+from camel.agents.tool_agents import BaseToolAgent
+
+
+class GoogleToolAgent(BaseToolAgent):
+    def __init__(
+        self,
+        name: str,
+        *args: Any,
+    ) -> None:
+        try:
+            from langchain.utilities import SerpAPIWrapper
```

Comment:
You are bringing in langchain as a dependency which is not reflected in requirements.txt

Reply:
In pyproject.toml, it is